src/photos.ts

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import {JSONPhoto} from './interfaces';
+
+export function photos(api_url: string, req: any, res: any) {
+    let page = req.query.page;
+    let camera = req.query.camera;
+    let paginationStart = req.query.paginationStart;
+    let paginationEnd = req.query.paginationEnd;
+    if (page && paginationStart || page && paginationEnd) {
+        res.send("Error: can't have both page and pagination params!");
+        return;
+    }
+    if (paginationStart && !paginationEnd || !paginationStart && paginationEnd) {
+        res.send("Error: must have both paginationStart and paginationEnd!");
+        return;
+    }
+    if (page) {
+        api_url += `&page=${page}`;
+    }
+    if (camera) {
+        api_url += `&camera=${req.query.camera}`;
+    }
+    if (paginationStart && paginationEnd) {
+        if (typeof paginationStart == "string" && typeof paginationEnd == "string") {
```

Comment:
why do we need this?

Reply:
the parseInt function used for going through the pages throws a type error, so I had to add that constraint

Reply:
I see, you don't have an else block, though - could you combine the 2 conditions (this and the one above)?